app-nexus-network/CHANGES.md

```diff
@@ -1,3 +1,7 @@
+Version 2.5.0
```

Comment:
We have been notified that we can by pass the canary process.

Please create a PR against master.

Thanks.

Reply:
Scratch this request.